CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 ## master
     - Support for chunk cache configuration at data set create/open
     - Support for std::array (unnested)
+    - Add direct data write shortcut 
```

Comment:
"Added AnnotateTraits::createAttribute and NodeTraits::createDataset overloads to create and initialize the data in a single call"